spec/design/skel/CNode_H.thy

```diff
@@ -10,13 +10,19 @@ theory CNode_H
 imports
   FaultMonad_H
   ThreadDecls_H
+  ArchRetypeDecls_H
   RetypeDecls_H
   TCBDecls_H
   CSpaceDecls_H
   EndpointDecls_H
   PSpaceFuns_H
 begin
 
+context begin interpretation Arch .
+requalify_consts
+  isArchMDBParentOf
+end
```

Comment:
arch-split docs now say to not do this, because this is very slow and I built infrastructure to avoid it.
Does `arch_requalify_consts` with an optional `(H)` not work for you here?

Reply:
Got it. If it got shoved into an Arch without global naming, you can do `requalify_consts Arch.isArchMDBParentOf`